cms/src/plugins/string-array/admin/src/components/Input/index.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { useState, useEffect } from 'react';
```

Comment:
```suggestion
import { useState } from 'react';
```
Remove no needed libs